src/api/index.js

```diff
@@ -162,9 +162,9 @@ export const verifyShare = ({ path, id }) =>
         : error.response.data.message
     })
 
-export const getProjects = () =>
+export const getProjects = (take = 0, skip = 0) =>
```

Comment:
`(take = 10, skip = 0)`

Reply:
we have standart value in redux/saga/ui/loadProjects
![image](https://user-images.githubusercontent.com/80621719/175777748-03ad00b1-db5e-43e8-ac2e-6cf4974b8424.png)